app/public/components/rct/rctData/pagesCellsSpecials.js

```diff
@@ -94,10 +94,10 @@ pagesCellsSpecials[pagesNames.dataPasses] = {
 };
 pagesCellsSpecials[pagesNames.mc] = {};
 
-const dateForametter = (sec) => {
-    const d = new Date(Number(sec));
-    const expected = -120;
-    d.setMinutes(d.getMinutes() - (expected - d.getTimezoneOffset()));
+const dateFormatter = (sec) => {
+    const cestOffset = 2 * 60 * 60 * 1000;
+    const localOffset = new Date().getTimezoneOffset() * 60 * 1000;
+    const d = new Date(Number(sec) + cestOffset + localOffset);
     return d.toLocaleString();
```

Comment:
I would also consider returning date an object of
```js
{
  date: d.toLocaleDateString();
  time: d.toLocaleTimeString();
}
```
for the sake of easier displaying these values with different styles, as the whole verbose date format is not needed anyway.